packages/field-highlighter/src/vaadin-user-tags.js

```diff
@@ -233,27 +232,6 @@ export class UserTags extends PolymerElement {
     return { added, removed };
   }
 
-  getChangedUsers(users, splices) {
-    const usersToAdd = [];
-    const usersToRemove = [];
-
-    splices.forEach((splice) => {
-      splice.removed.forEach((user) => {
-        usersToRemove.push(user);
-      });
-
-      for (let i = splice.addedCount - 1; i >= 0; i--) {
-        usersToAdd.push(users[splice.index + i]);
-      }
-    });
-
-    // Filter out users that are only moved
-    const addedUsers = usersToAdd.filter((u) => !usersToRemove.some((u2) => u.id === u2.id));
-    const removedUsers = usersToRemove.filter((u) => !usersToAdd.some((u2) => u.id === u2.id));
```

Comment:
IIRC we added this explicit `id` check since the Collaboration Kit calls `setUsers()` every time with a new array of objects, so the new comparison should probably be adjusted accordingly instead of using `.includes()`.

Reply:
Done, added a test to cover the case